v3.4/glfw/build.go

```diff
@@ -0,0 +1,57 @@
+package glfw
+
+/*
+#cgo CFLAGS: -Iglfw/include -D_GNU_SOURCE
```

Comment:
So then, `_GNU_SOURCE` is now globally set?

Reply:
If no response to https://github.com/go-gl/glfw/pull/400#issuecomment-2098661241, I'll just mark this as resolved.